torch/_inductor/mkldnn_lowerings.py

```diff
@@ -507,6 +507,9 @@ def qlinear_unary(
             algorithm,
             layout=None,
         ):
+            assert (
+                packed_weight.get_dtype() is torch.int8
+            ), "Only int8 weights are supported by oneDNN qlinear."
```

Comment:
Hi @Xia-Weiwen, I noticed that you removed this assert in the main branch. Would some future change necessitate its removal? Thanks!